arabic/unpacked/html.js

```diff
@@ -0,0 +1,90 @@
+MathJax.Hub.Config({
+  'HTML-CSS': {
+    undefinedFamily: 'Amiri',
```

Comment:
You are overriding the author's setting for `undefinedFamily`, here, or at least wiping out the default setting.  You might want to add your font to the existing ones, e.g.,
```
    undefinedFamily: 'Amiri,' + MathJax.Hub.config["HTML-CSS"].undefinedFamily,
```
to prepend your font to the ones already listed.  That way, you won't lose the undefined family used for missing characters that are *not* arabic ones.